### PR TITLE
add peer review modal search and link

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[speedup Q2] Activities: Peer review modals (5)](https://trello.com/c/6lSvc3o4/7228-speedup-q2-activities-peer-review-modals-5)

File: src/app/cdk/panel/panels/panels.component.ts [hunk 1]
```diff
@@ -19,9 +24,33 @@ export class PanelsComponent implements OnInit {
...
-  constructor() {}
+  constructor(
+    private _dialog: MatDialog,
+    private _platform: PlatformInfoService
+  ) {}
+
+  add() {
+    switch (this.type) {
+      case 'peer-review':
+        this.openModal(ModalPeerReviewsComponent)
```
Comment (leomendoza123) [inline src/app/cdk/panel/panels/panels.component.ts:35]: @DanielPalafox Maybe the modal should be an input of the component? Similar to what we do on the app-component

File: src/app/record/components/peer-reviews/modals/modal-peer-reviews/modal-peer-reviews.component.html [hunk 2]
```diff
@@ -25,5 +25,34 @@
...
+    <div class="row">
+      <div class="col orc-font-small-print">
+        <p i18n="@@shared.searchAndLink">
+          Search and Link wizards are our recommended way to populate your
+          record. They make adding works, funding and peer reviews simple and
+          save you time over updating your record manually. Select a platform
+          from the list below to start linking items to your record.
+        </p>
+        <a class="underline" href="">
+          More information about linking peer reviews to your ORCID record
```
Comment (leomendoza123) [inline src/app/record/components/peer-reviews/modals/modal-peer-reviews/modal-peer-reviews.component.html:39]: Require i18n tag

Comment (leomendoza123) [reply src/app/record/components/peer-reviews/modals/modal-peer-reviews/modal-peer-reviews.component.html:39]: also no HREF?

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: src/app/record/components/peer-reviews/modals/modal-peer-reviews/modal-peer-reviews.component.html (hunk 2 above)
Comment (DanielPalafox) [reply src/app/record/components/peer-reviews/modals/modal-peer-reviews/modal-peer-reviews.component.html:39]: ohh my bad! I will fix it

File: src/app/cdk/panel/panels/panels.component.ts (hunk 1 above)
Comment (DanielPalafox) [reply src/app/cdk/panel/panels/panels.component.ts:35]: Ok

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**